Makefile

```diff
@@ -316,9 +316,8 @@ define stringreplace
 	$(build_depsbindir)/stringreplace $$(strings -t x - $1 | grep '$2' | awk '{print $$1;}') '$3' 255 "$(call cygpath_w,$1)"
 endef
 
-install: $(build_depsbindir)/stringreplace
+install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html
```

Comment:
I guess we could list all files under `base` and `doc` as dependencies so that `make` compares their timestamps to that of the `html` directory. [Looks like this would do the trick](http://stackoverflow.com/questions/14289513/makefile-rule-that-depends-on-all-files-under-a-directory-including-within-subd). How does that sound?